scripts/webpack/webpack.common.js

```diff
@@ -48,6 +48,30 @@ module.exports = {
         __dirname,
         '../../public/app/overrides/components/AppSelector/Label'
       ),
+      '@webapp/components/LoadAppNames': path.resolve(
+        __dirname,
+        '../../public/app/components/LoadAppNames'
+      ),
+      '@webapp/components/Sidebar': path.resolve(
+        __dirname,
+        '../../public/app/components/Sidebar'
+      ),
+      '@webapp/components/Sidebar.module.css': path.resolve(
+        __dirname,
+        '../../public/app/components/Sidebar.module.css'
+      ),
+      '@webapp/components/SidebarTenant': path.resolve(
+        __dirname,
+        '../../public/app/components/SidebarTenant'
+      ),
+      '@webapp/components/SidebarTenant.module.css': path.resolve(
+        __dirname,
+        '../../public/app/components/SidebarTenant.module.css'
+      ),
+      '@webapp/components/TenantWall': path.resolve(
+        __dirname,
+        '../../public/app/components/TenantWall'
+      ),
```

Comment:
I think these are not needed, we can just add a global rule `@phlare/*` and import those as `@phlare/compoennts/MyComponent`.

Reply:
Updated! 

But do we need to get rid of the name `phlare` in the future?

Reply:
Thanks for your review!  💚 💙 💜 💛 ❤️

Reply:
>But do we need to get rid of the name phlare in the future?
we could rename to something else instead of "phlare", but the idea is to always have absolute imports, so that downstream projects (ie projects that just import this code) can override specific components. similar to what we do in this repo regarding `pyroscope-oss`

Reply:
Sounds good! Thanks!